content/docs/guides/vercel.md

```diff
@@ -57,7 +57,7 @@ To add the integration:
 
           The **Create a branch for your development environment** option creates a branch named `vercel-dev` and sets Vercel development environment variables for it. The `vercel-dev` branch is a clone of your project's primary branch (`main`) that you can modify without affecting data on your primary branch.
 
-          The **Automatically delete obsolete Neon branches** option automatically deletes a Neon preview branch when the git branch that triggered its creation is deleted.
+          The **Automatically delete obsolete Neon branches** option automatically deletes a Neon preview branch when the git branch that triggered its creation is merged or deleted.
```

Comment:
```suggestion
          With **Automatically delete obsolete Neon branches** enabled, Neon preview branches will be deleted whenever the git branch that triggered its creation is merged or deleted.
```

Reply:
Slight chance of possible confusion in the original text, unless I'm misunderstanding, enabling this feature will delete branches going forward, not existing branches. Not 100% sure this revised text eliminates the possible confusion.